cmd/remove.go

```diff
@@ -17,18 +17,23 @@ Example:
   $ shorten remove go
   $ shorten remove x29jzI8m`,
 	Run: func(cmd *cobra.Command, args []string) {
-		repoName := args[0]
 
-		repo, err := findRepo(repoName)
-		if err != nil {
-			fmt.Printf("Repository `%s` not found.\n", repoName)
-			return
+		// do input check before running command
+		if len(args) == 0 {
+			var repoName string
+			fmt.Print("Please input shortened URL you want to remove : ")
+			fmt.Scanln(&repoName)
+			args = append(args, repoName)
 		}
 
-		_, err = client.Repositories.Delete(ctx, repo.Owner, repo.Repo)
-		checkError(err)
+		// then do the repo removal
+		for _, arg := range args {
```

Comment:
Nice! Can we get this added this to the [`remove` help dialogue](https://github.com/kshvmdn/shorten/blob/master/cmd/remove.go#L13) (and an example too)? Just so users know that they can optionally provide a space-separated list of repos to remove.